### PR TITLE
Add support for WitsmlRisks

### PR DESCRIPTION
**Pull request by stigrus**
- Added support for WitsmlRisks

# Request Template Witsml Explorer
Fixes # (issue)

## Description
_Please include a summary of the change and which issue is fixed. List any dependencies that are required for this change._

...

## Type of change
_Put an x in the boxes that apply. You can also fill these out after creating the PR._

- [ ] Bugfix
- [x] New feature (non-breaking change which adds functionality)
- [ ] Enhancement of existing functionality
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Impacted Areas in Application
_List general components of the application that this PR will affect:_

*

# Checklist:
_Put an x in the boxes that are fulfilled._

- [ ] PR is related to an issue
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have used descriptive naming on components, functions and variables to avoid additional explanatory comments in the code
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

## Further comments
_If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc..._

### CODE REVIEW

**Review by lindsve — APPROVED** [review]
LGTM, but @haralde should also take a look at it

**Review by haralde — APPROVED** [review]
Looks good!

File: Src/Witsml/Data/WitsmlRisk.cs [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+        [XmlElement("subCategory")] public string SubCategory { get; set; }
+        [XmlElement("extendCategory")] public string ExtendCategory { get; set; }
+        [XmlElement("dTimStart")] public string DTimStart { get; set; }
+        [XmlElement("dTimEnd")] public string DTimEnd { get; set; }
+        [XmlElement("mdBitStart")] public string MdBitStart { get; set; }
+        [XmlElement("mdBitEnd")] public string MdBitEnd { get; set; }
+        [XmlElement("severityLevel")] public string SeverityLevel { get; set; }
+        [XmlElement("summary")] public string Summary { get; set; }
+        [XmlElement("details")] public string Details { get; set; }
+
```
Comment (steinsiv) [inline Src/Witsml/Data/WitsmlRisk.cs:26]: I noticed there was a property under Risk called `probabilityLevel` 
image
I do not know if it needs to be included? just thought I should mention it.
(from chapter 7.2 in `WITSML_STORE_API_Version_1.4.1.1.pdf`)

**Review by steinsiv — APPROVED** [review]
LGTM 👍 
I only had a little query on the `probabilityLevel` property in `WitsmlRisk.cs`

**Pull request merged**